src/SME.SGP.WebClient/src/paginas/CalendarioEscolar/Eventos/eventosForm.js

```diff
@@ -356,11 +378,15 @@ const EventosForm = ({ match }) => {
     setlistaCalendarioParaCopiar(eventos);
   };
 
-
   const onChangeTipoEvento = (evento, form) => {
     if (evento) {
-      const tipoEventoSelecionado = listaTipoEvento.find(item => item.id == evento)
-      if (tipoEventoSelecionado && String(tipoEventoSelecionado.descricao).toUpperCase() === 'FERIADO') {
+      const tipoEventoSelecionado = listaTipoEvento.find(
+        item => item.id == evento
+      );
+      if (
+        tipoEventoSelecionado &&
```

Comment:
não consegue botar essa comparação por código?

Reply:
Este trecho de código não foi implementado na resolução deste bug. A alteração se deve pelo fato do nosso plug-in de formatação ter colocado no padrão definido no projeto.